.golangci.yaml

```diff
@@ -33,6 +33,7 @@ linters:
     - goconst
     - golint
     - godot
+    - forbidigo
```

Comment:
`forbidgo` is profoundly useless. It literally bans certain regexes from your codebase - notable any use of `fmt.Print(f|ln)` (because I guess it's illegal to print to stdout?)